packages/block-editor/src/components/keyboard-shortcuts/index.js

```diff
@@ -93,6 +93,16 @@ function KeyboardShortcutsRegister() {
 			},
 		} );
 
+		registerShortcut( {
+			name: 'core/block-editor/multi-text-selection',
+			category: 'selection',
+			description: __( 'Select text between multiple blocks.' ),
```

Comment:
```suggestion
			description: __( 'Select text across multiple blocks.' ),
```

I think you could argue we are not selecting text **between** blocks but _across_ or _spanning_ blocks.

Reply:
I agree, this is a great suggestion.